test/test_grabs.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+
+from common_test_openrave import *
```

Comment:
@rschlaikjer Now `from common_test_openrave import *` is not allowed.

Well, I fixed.